packages/core/src/breadcrumb/breadcrumb.element.ts

```diff
@@ -0,0 +1,80 @@
+import { baseStyles, getCssPropertyValue, internalProperty } from '@cds/core/internal';
+import { html, LitElement } from 'lit-element';
+import { styles } from './breadcrumb.element.css.js';
+
+export const CdsBreadcrumbTagName = 'cds-breadcrumb';
+
+interface ItemEntry {
+  element: Element;
+  separator: Element;
+}
+
+/**
+ * ```typescript
+ * import '@cds/core/breadcrumb/register.js';
+ * ```
+ *
+ * ```html
+ * <cds-breadcrumb aria-label="breadcrumb">
+ *   <a href="/home" cds-text="link">Home</a>
+ *   <a href="/parent" cds-text="link">Parent page</a>
+ *   <span aria-current="page">Current page</span>
+ * </cds-breadcrumb>
+ * ```
+ *
+ * @beta
+ * @element cds-breadcrumb
+ * @cssprop --font-size
+ * @cssprop --font-weight
+ * @cssprop --letter-spacing
+ * @cssprop --gap - Gap between items. Valid values: xxs, xs, sm, md, lg, xl, xxl (default: sm)
+ * @cssprop --color
+ */
+export class CdsBreadcrumb extends LitElement {
+  @internalProperty({ type: String, reflect: true, attribute: 'role' })
+  protected role = 'navigation';
+
+  get gap() {
+    const gapValue = getCssPropertyValue('--gap', this);
+    return gapValue ? gapValue : 'sm';
```

Comment:
Using `gap` like this here is clever. But how do we know we are going to catch dynamic changes to the custom property and run an update? It would be great to see a test on this. The test would look something like...

1. set up a breadcrumb
2. add a classname to it or add a style value to it that changes `--gap` 
3. verify that the `ol` and `li` down below have their `gap:*` values updated in `cds-layout`

This is a bonus, imo. I think people changing the gap after the component is rendered is an edge case. It would be painful (relatively speaking) to update it on a window resize. And I can't think of any other immediate edge.

Still, it's a good to know sort of thing. If you have time, a test validating it works would be awesome. If not, could you add a `// TODO: ` here so we can remember to be aware?

Reply:
The gap does not change when changing the custom CSS property - @jeeyun any idea how to change this?

Reply:
@mathisscott brings up a valid point. I thought about this some more and think we should get rid of the `--gap`.  Since custom properties are commonly used for dynamic changes, I think using it in a way that violates user expectation is not ideal. 

An application could easily add more padding to a custom separator to achieve similar results:

```
<cds-breadcrumb aria-label="breadcrumb">
  <span cds-layout="p-x:lg" slot="cds-separator">🚘</span>
  <a href="#" cds-text="link">Home</a>
  <a href="#" cds-text="link">Parent page</a>
  <span aria-current="page">Current page</span>
</cds-breadcrumb>
 ```
 
Above code rendered: 
![Screen Shot 2021-04-27 at 4 25 39 PM](https://user-images.githubusercontent.com/1827742/116324592-45439000-a775-11eb-87ee-5de1ca5335ee.png)

Reply:
Removed the gap CSS property and adapted the example in the stories